test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -141,6 +142,38 @@ var _ = Describe("Mixedcpus", Ordered, func() {
 		})
 
 		When("workloads requests access for shared cpus", func() {
+			It("verify cpu load balancing still works with mixed cpus", func() {
+				rl := &corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("2"),
```

Comment:
The CI is falling because you need to ask for a single CPU here.
We only have 4 in total -
1 `reserved`
1 `shared`
1 used by infra pods
1 `isolated`

Reply:
I will update, but this test is bound to fail on D/S Environment . Though with full-pcpus-only: false  allows single cpus but this works only in the case of restricted  toplogy manager policy , In the case of single-numa-node the test case will fail on SMT Alignmnet error. 

```
          s: "failed to create pod. pod=performance-addon-operators-testing/test-l7rwc, podStatus={Failed [{PodReadyToStartContainers False 0001-01-01 00:00:00 +0000 UTC 2024-04-21 06:38:23 -0400 EDT  } {Initial
ized True 0001-01-01 00:00:00 +0000 UTC 2024-04-21 06:38:23 -0400 EDT  } {Ready False 0001-01-01 00:00:00 +0000 UTC 2024-04-21 06:38:23 -0400 EDT PodFailed } {ContainersReady False 0001-01-01 00:00:00 +0000 UTC 
2024-04-21 06:38:23 -0400 EDT PodFailed } {PodScheduled True 0001-01-01 00:00:00 +0000 UTC 2024-04-21 06:38:23 -0400 EDT  }] Pod was rejected: SMT Alignment Error: requested 1 cpus not multiple cpus per core = 2
 SMTAlignmentError  192.168.122.31 [{192.168.122.31}]  [] 2024-04-21 06:38:23 -0400 EDT [] [{test {nil nil &ContainerStateTerminated{ExitCode:137,Signal:0,Reason:ContainerStatusUnknown,Message:The container coul
d not be located when the pod was terminated,StartedAt:0001-01-01 00:00:00 +0000 UTC,FinishedAt:0001-01-01 00:00:00 +0000 UTC,ContainerID:,}} {nil nil nil} false 0 quay.io/openshift-kni/cnf-tests:4.14   0xc000a6
8864 map[] nil}] Guaranteed []  []} err=context deadline exceeded",   
```